code/src/App.css

```diff
@@ -0,0 +1,9 @@
+h2{
+  font-size: 30px;
+  margin: 12px;
+}
+
+h3{
+  font-size: 16px;
+  margin: 12px;
+}
```

Comment:
Is this a "general" css file? Then maybe some of the css from `Movie.css` should be extracted to here